static/src/stylesheets/module/reader-revenue/_epic.scss

```diff
@@ -1,51 +1,5 @@
 @import '../experiments/svg';
```

Comment:
a tiny amount of this css is still used for a message on the 'Today in Focus' page and the adblock message